mne_connectivity/spectral/epochs.py

```diff
@@ -1582,14 +1641,13 @@ def spectral_connectivity_epochs(data, names=None, method='coh', indices=None,
     con : array | list of array
         Computed connectivity measure(s). Either an instance of
         ``SpectralConnectivity`` or ``SpectroTemporalConnectivity``.
-        The shape of each connectivity dataset is either
-        (n_signals ** 2, n_freqs) mode: 'multitaper' or 'fourier'
-        (n_signals ** 2, n_freqs, n_times) mode: 'cwt_morlet'
-        when "indices" is None, or
-        (n_con, n_freqs) mode: 'multitaper' or 'fourier'
-        (n_con, n_freqs, n_times) mode: 'cwt_morlet'
-        when "indices" is specified and "n_con = len(indices[0])". If a
-        multivariate method is called "n_con = 1" even if "indices" is None.
+        The shape of each connectivity dataset is either:
+        (n_cons, n_freqs) mode: 'multitaper' or 'fourier'; or
+        (n_cons, n_freqs, n_times) mode: 'cwt_morlet'. When "indices" is None
+        and a bivariate method is called, "n_cons = n_signals ** 2", or if a
+        multivariate method is called "n_cons = 1". When "indices" is
+        specified, "n_con = len(indices[0])" for bivariate and multivariate
```

Comment:
```suggestion
        (n_cons, n_freqs, n_times) mode: 'cwt_morlet'. When "indices" is ``None``
        and a bivariate method is called, ``n_cons = n_signals ** 2``, or if a
        multivariate method is called ``n_cons = 1``. When "indices" is
        specified, ``n_con = len(indices[0])`` for bivariate and multivariate
```

Suggestion is just formatting.  The content also needs clarifying I think. Might be better as a bulleted list, like

```rst
The shape of the connectivity result will be:

- ``(n_cons, n_freqs)`` for multitaper or fourier modes
- ``(n_cons, n_freqs, n_times)`` for cwt_morlet mode
- ``(n_signals ** 2, n_freqs)`` for bivariate methods with ``indices=None``
- ``(1, n_freqs)`` for multivariate methods with ``indices=None``
- ``(len(indices[0]), n_freqs)`` for bivariate and multivariate methods when indices is supplied.

```

Reply:
How about this?
https://github.com/mne-tools/mne-connectivity/blob/402cfaa43b6f5827c3d44d262fa843ab3d849a07/mne_connectivity/spectral/epochs.py#L1644-L1652